kqueen_ui/asset/dynamic/scss/main.scss

```diff
@@ -115,31 +115,43 @@ a, input[type="checkbox"] {
   pointer-events: none;
 }
 
+.provisioner-pie {
+  line-height: 1.1;
+}
+
+// *****************************************************************************
+// Login
+// *****************************************************************************
+
 div.login {
   margin-top: 20%;
-  h3 {
-    margin: 5px 0px;
+  img {
+    margin-top: 35px;
+    width: 30%;
   }
-  ::-webkit-input-placeholder {
-    text-align: center;
+  form {
+    margin: 20px;
   }
-  :-moz-placeholder {
-    text-align: center;
+  .wrapper {
+    width: 70%;
+    position: relative;
+    padding: 0;
+    margin: auto;
   }
-  ::-moz-placeholder {
-    text-align: center;
+  .panel-footer {
+    font-size: 0.9em;
+    span {
+      margin-right: 14px;
```

Comment:
We don't need the margin for "Forgot your password?". Maybe it's better to add a class to "Need to sign up?" and add the margin for it, maybe even more than this, about 30-40px.